shtub.sh

```diff
@@ -139,6 +139,13 @@ _stub::set_all_methods() {
   done
 }
 
+# Asserts that the last command passed
+_stub::assert() {
```

Comment:
Note: I have no idea how to actually test this since we are using a built-in shpec testing construct which cannot actually be stubbed in any sane way.